### PR TITLE
Dynamic UI build + update to version 2.2.1.0

### PR DESCRIPTION
**Pull request by turkycat**

### CODE REVIEW

**Comment by ooeygui** [comment]
:+1:

File: remote-wiring-experience/MainPage.xaml.cs [hunk 1]
```diff
@@ -348,52 +353,51 @@ private void loadDigitalControls()
                 var text = new TextBlock();
                 text.HorizontalAlignment = HorizontalAlignment.Stretch;
                 text.VerticalAlignment = VerticalAlignment.Center;
-                text.Margin = new Thickness(0, 0, 0, 0);
-                text.Text = "Pin " + i;
+                text.Margin = new Thickness( 0, 0, 0, 0 );
+                text.Text = "Pin " + ( analogPins.Contains( digitalPins[i] ) ? "A" + ( digitalPins[i] - analogOffset ) : "" + digitalPins[i] );
```
Comment (mwmckee) [inline remote-wiring-experience/MainPage.xaml.cs]: This looks like it adds both analog and digital pins, but aren't analog pins loaded in the method below? (loadAnalogControls)

Comment (turkycat) [inline remote-wiring-experience/MainPage.xaml.cs]: Analog pins is always subset of digital pins.

Comment (mwmckee) [inline remote-wiring-experience/MainPage.xaml.cs]: Derp. Yup. Thanks!

**Comment by mwmckee** [comment]
Signoff

**Pull request merged**